docs/conf.py

```diff
@@ -0,0 +1,99 @@
+import re
+import sys
+import os
+
+
+project = "nextcord-ext-menus"
+copyright = "2021-present, Nextcord. 2015-2020, Danny (Rapptz)"
```

Comment:
Fixed incorrect copyright years
```suggestion
copyright = "2021-present, Nextcord. 2020-2021, Danny (Rapptz)"
```